src/main/java/org/alliancegenome/curation_api/model/entities/Association.java

```diff
@@ -19,8 +28,14 @@
 @Data @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
 @Inheritance(strategy = InheritanceType.JOINED)
 @Schema(name = "association", description = "Annotation class representing a disease annotation")
-@AGRCurationSchemaVersion(min="1.0.0", max=LinkMLSchemaConstants.LATEST_RELEASE, dependencies={AuditedObject.class})
-public class Association extends UniqueIdAuditedObject {
+@AGRCurationSchemaVersion(min="1.3.3", max=LinkMLSchemaConstants.LATEST_RELEASE, dependencies={AuditedObject.class})
+public class Association extends GeneratedAuditedObject {
 
+	@IndexedEmbedded(includeDepth = 2)
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@ManyToMany
+	@JsonView({View.FieldsAndLists.class})
+	private List<InformationContentEntity> evidence;
```

Comment:
Does all Associations have evidence? Maybe I am confused but I thought that was the point of an SlotAnnotation?

Reply:
Yep, Associations have evidence.  Only real difference from SlotAnnotations at this point is predicate, and separate ingest (rather than inlined) for Associations.